curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646bb182a47c3f0759416db7.md

```diff
@@ -0,0 +1,150 @@
+---
+id: 646bb182a47c3f0759416db7
+title: Step 28
+challengeType: 0
+dashedName: step-28
+---
+
+# --description--
+
+Javascript starts from this step. Use the `.getElementById()` method to get the `#current-date` element. Assign it to a `currentDateParagraph` variable.
```

Comment:
This should be your new step 1 once you remove all of the previous HTML and CSS steps. 

```suggestion
In this project, you will learn about the JavaScript Date object by building a date formatter.

All of the HTML and CSS for this project has been provided for you.

Start by getting the `#current-date` element with the `.getElementById()` method and assign it to a `const` variable called `currentDateParagraph`. 
```

Reply:
added